tools/filters/headWrapper.xml

```diff
@@ -4,11 +4,21 @@
     <edam_operation>operation_3695</edam_operation>
   </edam_operations>
   <command>
-perl '$__tool_directory__/headWrapper.pl' '$input' $lineNum '$out_file1'
+    set -eo pipefail;
```

Comment:
I guess we need to add a profile version here so that the non-zero exit code fails the job ?
We also need to bump the versio, add the tool versions to https://github.com/mvdbeek/galaxy/blob/926268ce6e1acd3db4e51ceb72e689899c545b9c/lib/galaxy/tools/__init__.py#L195 and wrap the command block in CDATA

Reply:
No problem. Should this be done as well for `Grep1`? What is the rule whether it should be included?

Reply:
If previously selected parameters don't change the output we should add it. If it does we should duplicate the tool in place.

Reply:
I think `Grep1` also fulfills these requirements. I already added it.